addons/huntir/Dialog.hpp

```diff
@@ -138,7 +138,7 @@ class ace_huntir_cam_dialog {
         x = __X + (2*0.8); y = __Y + 0.045;
         w = 0.05; h = 0.02;
         text = CSTRING(HELP);
-        action = "createDialog 'ace_huntir_help_dialog'";
+        action = QUOTE(createDialog QQGVAR(help_dialog));
```

Comment:
FYI:
you can also write:
`action = QUOTE(createDialog 'GVAR(help_dialog)');`

Macros are resolved inside single quote strings.